css/style.css

```diff
@@ -0,0 +1,309 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;  
+
+}
+
+body{
+    background-color: black;
+
+  }
+
+  /* Navegation bar */
+
+nav {
+    width: 100%;
+    position: fixed;
+    top: 0;
+    left: 0;
+    padding: 20px 8%;
+    display: flex;
+    align-items: center;
+    justify-content: space-between; 
+    font-family: 'Blanka', sans-serif;
+
+}
+
+nav ul li{
+    list-style: none;
+    display: inline-block;
+    margin-left: 45px;
+    
+}
+
+nav ul li a{
+    color: antiquewhite;
+    text-decoration: none;
+    font-size: 20px;
+
+}
+
+/* video*/
+.home-video{
+    position: relative;
+    width: 100%;
+    top: 0;
+    height: auto;
+     
+}
+
+@media (min-aspect-ratio: 16/9){
+    .home-video{
+        width: 100%;
+    }
+}
+
+@media (max-aspect-ratio: 16/9){
+    .home-video{
+        width: 100%; 
+    }
+}
+
+h1 {
+    font-family: 'Blanka', sans-serif;
+    color: antiquewhite;
+    text-align: center;                                           
+    font-size: 75px;
+    font-weight: bold;
+    margin: 100px;
+    padding-top: 50px;
+    padding-bottom: 50px;
+
+}
+
+/* about me section*/
+
+#about {
+    text-align: center;
+    color: white;
+    font-size: 20px;
+    margin-top: 50px;
+    padding: 30px 50px 100px 50px;
+
+}
+
+#about-me{ 
+    border-style: none;
+    text-align: justify;
+    margin-top: 100px;
+    margin-bottom: 100px;
+    margin-right: 200px;
+    margin-left: 140px;
+    font-family: Georgia, 'Times New Roman', Times, serif;
+    font-size: 25px;
+    height: 200px;
+    width: 50%;
+
+}
+
+img {
+    float:right;
+    width: 70px;
+    height:60px;
+
+}
+
+#img-riri {
+    float:right;
+    width: 300px;
+    height:300px;
+    margin-right: 180px;  
+
+}
+
+/* experience section */
+
+table {
+    padding: 30px 50px 100px 50px;
+    width: 70%;
+    margin-left: auto;
+    margin-right: auto;
+
+}
+
+table, th, td {
+  border-style: inset;
+  padding: 8px;
+
+}
+
+th {
+    font-family: 'Blanka', sans-serif;
+}
+
+table tr:nth-child(even){background-color: #4f4b4b;}
+
+table tr:hover {background-color: rgb(24, 24, 24);}
+
+table, th {
+  padding-top: 12px;
+  padding-bottom: 12px;
+  text-align: left;
+  background-color: rgb(24, 24, 24);
+  color: antiquewhite;
+  border: 5px double;
+  text-align: center;
+  font-size: 20px;
+
+}
+
+
+/* skill section */
+
+#skills {
+    padding: 30px 50px 100px 50px;
+    width: 100%;
+    margin-left: auto;
+    margin-right: auto;
+
+}
+
+.skl {
+    width: 18%;
+    border: 5px double;
+    color: antiquewhite;
+    font-size: 30px;
+    text-align: center;
+    display: inline-block;
+    padding: 16px;
+    margin-bottom: 0.5rem;
+    margin-left: 70px;
+    
+}
+
+/* leave a message section */
+form{
```

Comment:
Technically, the rubric for this assignment asks you to layout the form using FlexBox. So you would need a `display: flex` in this block and likely you'll want to have all the items laid out vertically, so also would need `flex-direction: column`. Then you can add spacing with the `gap` [property](https://www.w3schools.com/cssref/css3_pr_gap.php) which may allow you to remove all the `<br>` elements from the form element's HTML.

Reply:
would this be right?

form{
    width: 25em;
    max-width: 90%;
    margin: 0 auto;
    font-family: 'Blanka', sans-serif;
    font-size: 15px;
    color: white;
    border-radius: 5px;
    background-color: #222222;
    display: flex;
    flex-direction: column;
    gap: 10px;
    padding: 20px;

}  
       
input[type=text], select {
    width: 100%;
    padding: 12px 20px;
    margin: 5px 0;
    border: 2px solid #ccc;
    border-radius: 4px;
    box-sizing: border-box;
}
     
input[type=email], select {
    width: 100%;
    padding: 12px 20px;
    margin: 5px 0;
    border: 2px solid #ccc;
    border-radius: 4px;
    box-sizing: border-box;
    
}
     
textarea[type=message] {
    width: 100%;
    padding: 20px 20px;
    margin: 5px 0;
    border: 2px solid #ccc;
    border-radius: 4px;
    box-sizing: border-box;

}
     
button{
    font-family: 'Blanka', sans-serif;
    font-size: 15px;
    background-color: rgb(24, 24, 24);
    border: none;
    border-radius: 4px;
    color: white;
    padding: 16px 32px;
    margin: 10px 2px;
    cursor: pointer;    

}

Reply:
Yep, I think the only CSS section that needs to change is `form {}` . What you have in your comment looks perfect :+1: